db/migrate/20170720194715_create_middleware_jdr_reports.rb

```diff
@@ -0,0 +1,12 @@
+class CreateMiddlewareJdrReports < ActiveRecord::Migration[5.0]
+  def change
+    create_table :middleware_jdr_reports do |t|
```

Comment:
BTW, do you expect a generic model in main repo? Right now, I only have a model in this PR: ManageIQ/manageiq-providers-hawkular#30

(May be you also know the answer to https://github.com/ManageIQ/manageiq-providers-hawkular/pull/30#issuecomment-319691445 on that same PR)